frontend/src/employee-frontend/state/invoicing-ui.tsx

```diff
@@ -88,6 +91,19 @@ interface InvoiceSearchFilterState {
   clearSearchFilters: () => void
 }
 
+export interface IncomeStatementSearchFilters {
+  area: string[]
+  providerTypes: ProviderType[]
+  sentStartDate: LocalDate | undefined
+  sentEndDate: LocalDate | undefined
+}
+
+interface InvoiceStatementSearchFilterState {
```

Comment:
I think the name of this interface should be `IncomeStatementSearchFilterState`. We don't have a concept called "invoice statement".